grow/documents/document.py

```diff
@@ -176,6 +176,12 @@ def collection_path(self):
         """The pod path relative to the collection path."""
         return self.pod_path[len(self.collection.pod_path):]
 
+    @property
+    def collection_path_clean(self):
```

Comment:
Is "collection_path" the best name? It feels like "collection_path" would refer to the collection's path, i.e. `/content/pages/` in the tests below.

Reply:
I'm open to other suggestions. `collection_sub_path`? I'm not sure what else to call it. Basically it is the pod path minus the collection path.

so in the `/content/pages/` collection a `pod_path` of `/content/pages/sub/index.yaml` would be `/sub/index.yaml`. It is the path inside of the collection... but I wasn't sure what to call it.

Reply:
Renamed it to `collection_sub_path` I think that matches it better.